### ISSUE TITLE
ci_scripts duplicates?

### ISSUE THREAD

**Issue by shukon** [automl/CAVE#282]
Are `ci_scripts/install.sh` and `ci_scripts/push_doc.sh` duplicates / still needed?

**Comment by AndreBiedenkapp** [comment]
I am unsure. Will have to check.

**Comment by shukon** [comment]
Yeah... well I can actually just delete them from dev and see what happens ;)
